RBDM/Debug/RBDM.Build.CppClean.log

```diff
@@ -0,0 +1,18 @@
+c:\users\vitaliy.pyza\rdbm\rdm\rbdm\debug\vc120.pdb
```

Comment:
Please, add this file to gitignore as all obj files that are above this comment.